<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-75OLIUH" name="amazon.TestSetAmazon" tests="6" failures="1" timestamp="22 Dec 2017 02:13:12 GMT" time="989.258" errors="0">
  <testcase name="TestCase_106_UpAndDown" time="56.430" classname="amazon.TestSetAmazon"/>
  <testcase name="TestCase_105" time="79.073" classname="amazon.TestSetAmazon"/>
  <testcase name="TC_102" time="671.193" classname="amazon.TestSetAmazon"/>
  <testcase name="SearchOnLandingPage" time="100.854" classname="amazon.TestSetAmazon"/>
  <testcase name="TestCase_104" time="58.696" classname="amazon.TestSetAmazon"/>
  <testcase name="TestCase_103" time="23.012" classname="amazon.TestSetAmazon">
    <failure type="junit.framework.ComparisonFailure" message="expected:&lt;[Pass]&gt; but was:&lt;[Fail]&gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<[Pass]> but was:<[Fail]>
	at junit.framework.Assert.assertEquals(Assert.java:100)
	at junit.framework.Assert.assertEquals(Assert.java:107)
	at amazon.TestSetAmazon.TestCase_103(TestSetAmazon.java:134)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- TestCase_103 -->
</testsuite> <!-- amazon.TestSetAmazon -->
